src/cmds/docs/index.ts

```diff
@@ -67,7 +76,17 @@ export default class DocsCommand extends Command {
 
     Command.debug(`number of files: ${files.length}`);
 
-    if (!files.length) {
+    const changes: string[] = [];
+    if (deleteMissing) {
+      const docs = await getDocs(key, selectedVersion);
```

Comment:
Could you add a `warn` statement above this line that warns the user about what they're about to do? Using this function:

https://github.com/readmeio/rdme/blob/0292a09eabc9e0beb0e982f6f24bed4c2f757a41/src/lib/logger.ts#L27-L32

Reply:
Done (using `Command.warn`)